### ISSUE TITLE
2 checking to-dos

### ISSUE THREAD

**Issue by supernova-97** [supernova-97/Capstone-Project#2]
# User Story 2 "checking to-dos as "Done""

## Value proposition
As a user
I want to be able to check to-dos as "done"
to be able to see what's already been done

![Image](https://user-images.githubusercontent.com/123933016/227457800-4197491d-b73b-438c-b12d-72b619d397df.png)

## Acceptance Criteria: 
- [ ] to-dos that are done will be grey and crossed out

## Tasks:
- [ ] do styling for done tasks

**Comment by Gblur** [comment]
So its about checking to-dos as done?
In this case I would also add a task like "create a handler function for changing style to specific items"

**Comment by supernova-97** [comment]
@Gblur great idea, I'll add it!

**Comment by PReichetanz** [comment]
Question: Can I undone a done todo? 🤯
If so, you should add another AC similar to this:
- If done todo is clicked again, it gets its default state again (i.e. it is an item which is still to be done).

**Comment by supernova-97** [comment]
@PReichetanz yes of course! I wanted to do that in the next story because I thought it was getting to big.
But I guess I can just toggle it and include it in this story then.

**Comment by salaos** [comment]
Looks good now - nothing to add! :)
(Just one thought: The "done"-Elements can move automatically to the end of the list, when clicked but maybe not for this user story)
